detail.html

```diff
@@ -0,0 +1,32 @@
+<%inherit file="app_base.htm" />
+
+
+<%block name="content">
+
+<h1 class="text-center">${ product.name | h }</h1>
+<div class="product_detail_flexbox">
+    <ul class="product_detail_list">
+        <%
+        images = product.image_urls()
+        %>
+        %for img in images:
+        <li><a href="#"><img class="product_detail_images" src="${ img }" alt="${ product.name }"></a></li>
+        %endfor
+    </ul>
+    <div>
+        <div class="product_detail_image">
+            <img id="main" src="${ product.image_url() }" alt="${ product.name }">
```

Comment:
You should probably htmlescape any text you're bringing down from the database (e.g. ${ product.name | h}

Reply:
Yes, I forget to do that here. I will go ahead and add that in.